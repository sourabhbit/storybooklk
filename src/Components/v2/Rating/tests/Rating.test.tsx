import { render, screen } from '@testing-library/react'
import NumericalRating from '../index'

const props = {
	rating: 4.5,
}

describe('<NumericalRating />', () => {
	let wrapper: any = null

	beforeEach(() => {
		const { asFragment } = render(<NumericalRating {...props} />)
		wrapper = asFragment
	})

	afterAll(() => {
		wrapper = null
	})

	it('Should take a snapshot', () => {
		expect(wrapper()).toMatchSnapshot()
	})
	it('Should render Numerical Rating component', () => {
		const container = screen.getByTestId('rating-test')
		expect(container).toBeInTheDocument()
	})
})
