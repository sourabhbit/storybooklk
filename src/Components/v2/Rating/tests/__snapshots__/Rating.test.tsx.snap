// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NumericalRating /> Should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvtL cPrLic"
    data-testid="rating-test"
  >
    <img
      alt=""
      class="sc-gsDKAQ jwEqgZ"
    />
    <div
      class="sc-dkPtRN eJzovV"
    >
      4.5
    </div>
  </div>
</DocumentFragment>
`;
